src/server/constants.ts

```diff
@@ -43,7 +43,7 @@ export const GENERATE_TYPES_DETECT_ONE_TO_ONE_RELATIONSHIPS =
 
 export const DEFAULT_POOL_CONFIG: PoolConfig = {
   max: 1,
-  connectionTimeoutMillis: PG_CONN_TIMEOUT_SECS * 1000,
```

Comment:
Removing this results in the default of 0 taking over - which is **not** what we want.